whiskey.yml

```diff
@@ -227,7 +212,6 @@ Build:
             - "Test/ConvertTo-WhiskeySemanticVersion.Tests.ps1"
             - "Test/CopyFile.Tests.ps1"
             - "Test/Delete.Tests.ps1"
-            - "Test/Exec.Tests.ps1"

```

Comment:
I'm starting to think it might be easier to include the Pester version in the test script name, e.g. `Exec.Tests.pester4.ps1`, so that we don't have to maintain this long list of exclusions. Then we just have a single task for each version with a wildcard, e.g. `Test/*.Tests.pester5.ps1` & `Test/*.Tests.pester4.ps1`.

Reply:
Then we would have to rename files when we switch versions and I think I prefer a longer commit history wihtout file renames over short lists in the whiskey.yml file.

Reply:
That's fair.